homeassistant/components/tessie/coordinator.py

```diff
@@ -55,6 +55,8 @@ async def _async_update_data(self) -> dict[str, Any]:
             if e.status == HTTPStatus.UNAUTHORIZED:
                 # Auth Token is no longer valid
                 raise ConfigEntryAuthFailed from e
+            if e.status == HTTPStatus.REQUEST_TIMEOUT:
+                raise UpdateFailed(f"Vehicle {self.vin} has no connectivity") from e
```

Comment:
I saw your message on Discord. This will indeed log an error everytime it loses connectivity. Another solution would be to return `dict[str, Any] | None` here and have all entities handle getting no data. They can turn unknown or unavailable (I think I prefer unknown as you don't know the state).

Reply:
Unknown makes more sense, I agree.